src/online2/online-nnet3-decoding.h

```diff
@@ -0,0 +1,132 @@
+// online2/online-nnet3-decoding.h
+
+// Copyright 2014  Johns Hopkins University (author: Daniel Povey)
```

Comment:
oh sorry I misunderstood.  I thought it was in nnet2.

On Fri, Apr 15, 2016 at 12:53 PM, Ilya Platonov <notifications@github.com>
wrote:

> In src/online2/online-nnet3-decoding.h
> <https://github.com/kaldi-asr/kaldi/pull/666#discussion_r59931875>:
>
> > @@ -0,0 +1,132 @@
> > +// online2/online-nnet3-decoding.h
> > +
> > +// Copyright 2014  Johns Hopkins University (author: Daniel Povey)
>
> online-nnet3-decoding.h is in online2 because online-nnet2-decoding.h is
> online2 as well.
>
> If I move this, then nnet3 will depend on online2, is this ok?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/666/files/97e675fffbaeb59db76c78842ca14e6c2ae58230#r59931875>
>